apps/native/test/main.c

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *******************************************************************************/
+
+/* Included Files. */
+#include <stdio.h>
+#include <orchestration.h>
+#include <pthread.h>
+#include <unistd.h>
+
+/* Private Data. */
+/* All static data definitions appear here. */
+RequestServiceInfo rsi = { 
+  .ExecutionType  = "native",
+  .ExeCmd         = "ls"
```

Comment:
```suggestion
    .ExeCmd         = "ls"
```

Reply:
@tdrozdovsky Please revise the code indentation. Above two recommendations are just some of the examples.

Reply:
We are using Plugin : gofmt.